### PR TITLE
Add zanneth's personal site (attempt 2)

### PR DESCRIPTION
**Pull request by zanneth**
Webring link is on the bottom left corner of the website, in the footer: https://zanneth.com

### CODE REVIEW

File: scripts/sites.js [hunk 1]
```diff
@@ -93,5 +93,6 @@ const sites = [
   { url: 'https://notes.stuartpb.com/', title: 'notes.stuartpb.com', type: 'wiki', author: 'stuartpb', contact: 's@stuartpb.com', feed: 'https://twtxt.stuartpb.com/xxiivv.txt' },
   { url: 'http://xxiii.co', title: 'xxiii', type: 'wiki', author: 'serocell', contact: 'psignal@s900.net', rss: 'http://serocell.com/feeds/serocell.xml' },
   { url: 'https://kor.nz', title: 'kor', type: 'wiki', author: 'kormyen', contact: 'h@kor.nz' },
-  { url: 'https://lublin.se', author: 'quite', contact: 'quite@hack.org', feed: 'https://lublin.se/twtxt.txt' }
+  { url: 'https://lublin.se', author: 'quite', contact: 'quite@hack.org', feed: 'https://lublin.se/twtxt.txt' },
+  { url: 'https://zanneth.com', title: 'zanneth.com', author: 'zanneth', type: 'blog', contact: 'root@zanneth.com' },
```
Comment (neauoire) [inline scripts/sites.js]: Remove trailing comma.

Comment (zanneth) [reply scripts/sites.js]: Thy deed is done.

**Comment by neauoire** [comment]
Welcome to the webring!

**Comment by zanneth** [comment]
Thanks!!

**Pull request merged**